ad-bank.md

```diff
@@ -0,0 +1,39 @@
+PostHog Ad Bank
+<hr/>
+
+## POSTHOG SELF-HOSTED
+
+* PostHog: product analytics software that’s fast, powerful, and easy to use - all completely self-hosted for maximum SOC 2 & HIPAA compliance. See it in action at posthog.com/product
```

Comment:
I'm confirming this, but little wary on this point too. To my knowledge we haven't completed our SOC 2 compliance yet and I couldn't find any information about HIPAA. In any case, my understanding is that these certifications mainly apply to the cloud product and that, in self-hosted instances, it's dependent on the compliance of the original environment. 'Easy compliance' is something we could stretch to, but claiming 'Maximum compliance' feels like a misleading security claim.

Reply:
Saw it mentioned on [the homepage](https://posthog.com).

Reply:
Basically @joethreepwood you are right - we have neither SOC 2 (on pause but mostly done) nor HIPAA (not started at all). It would only really make sense for us to claim either certification for PostHog Cloud. If a customer is self-hosting, then really it is more relevant to know whether or not the customer themselves are SOC 2 or HIPAA compliant. 

@shehuphd the mention on the Homepage that PostHog self-host "Makes HIPAA and SOC2 audits a breeze when you host on your own stack" means that the customer doesn't have to worry about sending data outside their infrastructure for the purposes of their own audits (not because PostHog is HIPAA/SOC2-compliant).